src/main/java/fr/novia/zaproxyplugin/ZAProxy.java

```diff
@@ -768,25 +854,29 @@ private void checkParams(AbstractBuild<?, ?> build, BuildListener listener)
 			throw new IllegalArgumentException("zapProxy Port is less than 0");
 		} else
 			listener.getLogger().println("zapProxyPort = " + zapProxyPort);
-
-		if(jiraBaseURL.equals(null)) {
-			throw new IllegalArgumentException("Jira Base URL not Found");
-		} else
-			listener.getLogger().println("jiraBaseURL = " + jiraBaseURL);
-
-		if(jiraUserName.equals(null)) {
-			throw new IllegalArgumentException("Jira User name not Found");
-		} else
-			listener.getLogger().println("jiraUserName = " + jiraUserName);
-
-		if(jiraPassword.equals(null)) {
-			throw new IllegalArgumentException("Jira password not Found");
-		} else {
-			String pass = "";
-			for (int i = 0; i < jiraPassword.length(); i++) {
-				pass += "*";
+		
+		//create jira is enabled
+		if(getcreateJiras()==true){
+			
+			if(jiraBaseURL.equals(null)) {
```

Comment:
This (and some of the following statements) leads to a NullPointerException if the user does not define the JIRA's base URL but enables the option "Create Jira Issues", following stack trace:
```
ERROR: java.lang.NullPointerException
	at fr.novia.zaproxyplugin.ZAProxy.checkParams(ZAProxy.java:861)
	at fr.novia.zaproxyplugin.ZAProxy.startZAP(ZAProxy.java:899)
	at fr.novia.zaproxyplugin.ZAProxyBuilder.perform(ZAProxyBuilder.java:194)
	at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
	at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:782)
	at hudson.model.Build$BuildExecution.build(Build.java:205)
	at hudson.model.Build$BuildExecution.doRun(Build.java:162)
	at hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:534)
	at hudson.model.Run.execute(Run.java:1738)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:98)
	at hudson.model.Executor.run(Executor.java:410)
```

The issue is less problematic with the changes purposed in this pull request, since it's only executed if the option is enabled (currently it leads to NullPointerException even if the option is not enabled).